src/Nancy.Tests/Unit/Diagnostics/DefaultRequestTraceFactoryFixture.cs

```diff
@@ -62,10 +62,7 @@ public void Should_initialize_case_insensitive_items_collection_when_case_sensit
                 trace.Items.Add("foo", "bar");
 
                 // Then
-                Assert.DoesNotThrow(() =>
-                {
-                    var value = trace.Items["FOO"];
-                });
+                var value = trace.Items["FOO"];

```

Comment:
@NancyFx/most-valued-minions please have a read of this https://github.com/xunit/xunit/issues/188 . Are we happy with leaving this as is, or would you prefer we used `Record.Exception` as suggested here https://github.com/xunit/xunit/issues/188#issuecomment-86584802 ?